File: app/travel-together-fe/src/models/Comment.ts
export class Comment {
  content: string;
  userIdComment: string;
  userAvt: string;

  constructor(content: string, userIdComment: string, userAvt: string) {
    this.content = content;
    this.userIdComment = userIdComment;
    this.userAvt = userAvt;
  }
}

File: back-end/config-service/src/main/java/com/traveltogether/configservice/document/Comment.java
package com.traveltogether.configservice.document;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Document(collection = "comment")
public class Comment extends BaseDocument<Comment> {

    private String content;

    private String userIdComment;

    private String userAvt;

    private String fullName;

    private Boolean isLocalGuide;

}
